documentation/crds_users_guide/source/library_use.rst

```diff
@@ -181,15 +181,15 @@ crds.get_default_context()
 ..........................
 
 `get_default_context()` returns the name of the pipeline mapping which is 
-currently the Latest Context
+currently the Build Context
```

Comment:
Same comment as above. `get_default_context` only defaults to build if the observatory is JWST. Otherwise it defaults to latest.